.github/workflows/auto-merge.yml

```diff
@@ -0,0 +1,16 @@
+name: auto-merge
+
+on:
+  pull_request_target:
+
+jobs:
+  auto-merge:
+    runs-on: ubuntu-latest
+    if: github.actor == 'dependabot[bot]'
+    steps:
+      - uses: ahmadnassri/action-dependabot-auto-merge@v2.6
+        with:
+          github-token: ${{ secrets.AUTOMERGE_TOKEN }}
```

Comment:
I believe we use `GH_TOKEN` everywhere else, but I think something like `PAT_TOKEN` would be even more descriptive. Thoughts?

Reply:
I just ctl+c ctrl+v'd from https://github.com/mdn/content/blob/main/.github/workflows/auto-merge.yml but I've no objection to `PAT_TOKEN` or `PA_TOKEN`